plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/utils/SwipeUtils.java

```diff
@@ -119,7 +120,9 @@ private static void swipe(Point location, Dimension size, DirectionStrategy dire
         int x = location.getX();
         int y = location.getY();
         LOG.debug("Swipe parameters: location {}, dimension {}, direction {}, time {}", location, size, direction, time);
-        appiumDriver.swipe(x + startx, y + starty, x + endx, y + endy, time);
+//        appiumDriver.swipe(x + startx, y + starty, x + endx, y + endy, time);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)